components/QuestionNotificationsToggleExplanationModal.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Modal, ModalHeader, ModalBody, ModalFooter, Button } from 'reactstrap'
+
+const QuestionNotificationsToggleExplanationModal = props => (
+  <Modal isOpen={props.isOpen} toggle={props.toggle}>
+    <ModalHeader>Question notifications</ModalHeader>
+    <ModalBody>
+      <h5>For students</h5>
+      <p>
+        If you enable notificaitons, you&apos;ll get notifications when your
+        question is being answered.
+      </p>
+      <h5>For course staff/admins</h5>
```

Comment:
Could these be wrapped and be shown based on user role (course staff vs not course staff)?

Reply:
I had a discussion with Nathan, and we agreed that probably it is a better choice to just show both messages because it's hard to handle edge cases properly when distinguishing roles. For example, Nathan is an admin, but also student in CS 461. But it's definitely better if we can handle the role more properly.

Reply:
ahhh, makes sense